code/game/machinery/cryo.dm

```diff
@@ -430,6 +438,14 @@
 	for(var/atom/movable/A in contents - component_parts - list(beaker))
 		A.forceMove(get_step(loc, SOUTH))
 
+/obj/machinery/atmospherics/unary/cryo_cell/proc/auto_eject(eject_healthy, eject_dead)
+	on = FALSE
+	go_out()
+	if(eject_healthy)
```

Comment:
Maybe use a flag here instead of two exclusive bools. If both are true only eject_healthy will get through